### ISSUE TITLE
Hello Panorama crashed after PLControlTypeSupportedNone;

### ISSUE THREAD

**Issue by GoogleCodeExporter** [niltsh/panoramagl#5]
```
What steps will reproduce the problem?
1. set plView.controlTypeSupported = PLControlTypeSupportedZoom;
2. start Hello Panorama
3.

What is the expected output? What do you see instead?
No Controls :-)   crashed

What version of the product are you using? On what operating system?
tested on iPad 3.2 and iOS 4.0.1 2G/3GS

Please provide any additional information below.


```

Original issue reported on code.google.com by `sd8...@googlemail.com` on 11 Aug 2010 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
Yes, I saw the same problem too. I think current panoramagl is really not 
optimized for ipad, it only works on ipad, but that's all
```

Original comment by `sha...@gmail.com` on 12 Aug 2010 at 4:57

**Comment by GoogleCodeExporter** [comment]
```
Yes, but it´s the same issue on iPhone and on iPhone Simulator :-)
```

Original comment by `sd8...@googlemail.com` on 12 Aug 2010 at 8:50

**Comment by GoogleCodeExporter** [comment]
```
It's an autorelease problem. The button to change (hide) is already released.
Go to PLControl.h/m and make the member button a property with retain attribute 
and change the line:

button = [UIButton buttonWithType:UIButtonTypeCustom];
to 
self.button = [UIButton buttonWithType:UIButtonTypeCustom];

That should fix the problem.
```

Original comment by `christia...@gmail.com` on 29 Nov 2010 at 8:12

**Comment by GoogleCodeExporter** [comment]
```
thanks for this bugfix

I don´t use the buttons so I commented all out with buttons, but this is the 
better way ;)
```

Original comment by `sd8...@googlemail.com` on 29 Nov 2010 at 10:34

**Comment by GoogleCodeExporter** [comment]
```
The memory leaks problems were fixed in 0.1 version and controls are not 
supported, if anybody want to use controls can add a subview in PLView.
```

Original comment by `javbae...@gmail.com` on 5 Jul 2012 at 3:56
- Changed state: **Fixed**
